src/reducers/movies.js

```diff
@@ -19,6 +19,21 @@ function movies(state = initialState.movies, action) {
         data: [],
         status: ACTION_TYPES.LOAD_MOVIES_ERROR
       };
+    case ACTION_TYPES.SELECT_MOVIE:
+      return {
+        ...state,
+        selectedMovie: action.movie
+      };
+    case ACTION_TYPES.LOAD_MOVIE_DETAILS_SUCCESS:
+      return {
+        ...state,
+        selectedMovie: [action.movie.data]
```

Comment:
I think there is no need to wrap `action.movie.data` in array.

Reply:
Fixed. Array is not needed. Object is sufficient.